managedtenants/data/metadata.schema.yaml

```diff
@@ -210,6 +210,30 @@ properties:
       secretNamespace:
         type: string
         pattern: ^[A-Za-z0-9][A-Za-z0-9-]{0,60}[A-Za-z0-9]$
+  deadmanssnitch:
```

Comment:
Please add some `description: ` field to help developers and tenants. They will be rendered as markdown as well: https://github.com/mt-sre/managed-tenants-cli/blob/main/docs/tenants/zz_schema_generated.md